tests/integration/lldp_test.py

```diff
@@ -328,33 +333,9 @@ def lldp_enabled(ifstate):
 def _send_lldp_packet():
     test_dir = os.path.dirname(os.path.realpath(__file__))
     cmdlib.exec_cmd(
-        (
-            "tcpreplay",
-            "--intf1=lldptestpeer",
-            f"{test_dir}/test_captures/lldp.pcap",
-        ),
+        f"ip netns exec {LLDP_TEST_NS} "
+        f"tcpreplay --intf1={LLDPTEST_PEER} "
+        f"{test_dir}/test_captures/lldp.pcap".split(),
```

Comment:
Why are you adding a split() here? Is this needed?

Reply:
The `exec_cmd()` need a list/tumple. In stead ugly place list here, a simple split() can do the magic.

Reply:
Oh, I see. I didn't notice it was a big string. Thanks for this, we should use it everywhere :-)